Admin news list

PhucJoseph opened the pull request.

Commit:
clear

Commit:
clear

Commit:
clean redux

Commit:
fix

Commit:
clear

Commit:
clear

mugetsu0422 commented on src/app/admin/news/newsSclice.js:
sai chính tả kìa ô, file tên phải là newsSlice

Commit:
clear

Commit:
clear

mugetsu0422 commented on src/app/admin/news/page.tsx:
```diff
@@ -1,9 +1,139 @@
+
+  if (pages > 0) {
+    setPages(1)
+  }
```
```suggestion
```
để v render vô hạn à

mugetsu0422 committed:
call api

mugetsu0422 committed:
Merge branch 'development' into feature/MP-admin-news-list

mugetsu0422 committed:
adjust UI and update create. edit

In admin news list:
 add create and reset button
In create and edit page:
 add summary input

mugetsu0422 committed:
adjust UI
